src/game_api/entity_db.hpp

```diff
@@ -99,6 +99,9 @@ struct EntityDB
     float default_special_offsetx;
     float default_special_offsety;
     uint8_t init;
+
+    EntityDB(const EntityDB& other) = default;
```

Comment:
This won't copy animations and in fact probably crashes with animations, but that's for someone else to finish. Works great for wallclimbing though:

```lua
E.type = EntityDB:new(E.type) -- create a new type with the copy constructor
E.type.properties_flags = set_flag(E.type.properties_flags, 6) -- set only this entity climbable
```

Reply:
Why will this not copy animations?